src/api/itemExport.ts

```diff
@@ -3,15 +3,17 @@ import { UUID } from '@graasp/sdk';
 import { buildExportItemRoute } from '../routes.js';
 import { PartialQueryConfigForApi } from '../types.js';
 
-/* eslint-disable import/prefer-default-export */
 export const exportItem = async (
   id: UUID,
   { API_HOST, axios }: PartialQueryConfigForApi,
 ) =>
-  // options?: { public: boolean },
   axios
     .get<Blob>(`${API_HOST}/${buildExportItemRoute(id)}`, {
       method: 'GET',
       responseType: 'blob',
     })
-    .then(({ data }) => data);
+    .then(({ data, headers }) => {
+      const name =
+        headers['content-disposition']?.split('"')?.[1] ?? 'export-file';
```

Comment:
Split on quotes ?

Reply:
Content is usually: `filename="name.png"` so it gets the given filename. I added comments.